lib/msal-browser/docs/logging.md

```diff
@@ -59,5 +59,24 @@ An example usage in a sample can be accessed [here](https://github.com/AzureAD/m
 
     ![browser console](./images/BrowserLogEnablement.png)
 
+## Override log level and PII setting
+
+These are the steps to override MSAL log level and PII settings to troubleshoot errors in non-dev environments:
+
+### Navigate to session storage
+
+1. Open developer tools by pressing F12
+2. Navigate to `Application` tab
+3. Click `Storage` and expand `Session Storage`
+4. Select target domain
+
+Additional details can be found [here](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/storage/sessionstorage).
+
+### Override log level
+
+Add `msal.browser.log.level` key to `Session Storage`, set it's value to the desired log level (`Verbose`, for example) and refresh the page.
+Check `LogLevel` enum for the available options [here](../../msal-common/src/logger/Logger.ts).
 
+### Override PII log setting
 
+Add `msal.browser.log.pii` key to `Session Storage`, set it's value to `true` or `false` and refresh the page.
```

Comment:
We need more steps here.  This should be something we can refer folks to for log collection and troubleshooting.  Something along the lines of:
`Once you have enabled Verbose logging and (optionally) enabled PII logging, refresh the page and retry the sign-in operation.  The captured logs can be found *here*.  Please review them to ensure there is no sensitive data included in them before sharing them with the team requesting them.`

Reply:
Updated docs, thank you for the suggestion.